client/src/pages/Home.css

```diff
@@ -6,7 +6,7 @@
 }
 
 body {
-	font-family: Arial, sans-serif;
+	font-family: Arial, sans-serif; 
```

Comment:
Could you please remove everything from this file that would be suitable to live in global styles.
Please replace colors with css vars
Please fix formatting.

Reply:
done